src/datachain/catalog/catalog.py

```diff
@@ -1124,7 +1124,7 @@ def get_dataset_with_remote_fallback(
             raise DatasetNotFoundError(
                 f"Dataset {name}"
                 + (f" version {version} " if version else " ")
-                + "not found"
+                + f"not found in namespace {namespace_name} and project {project_name}"
```

Comment:
will it look like `local and project local` in some (all) cases?

Reply:
I think we should try to avoid `local` / `local` when people deal with local setup. May we should have a helper to convert all these errors into something human readable - e.g. skip local completely, or say something like "Can't find in local namespace" ...